src/main/java/domain/LadderGame.java

```diff
@@ -0,0 +1,26 @@
+package domain;
+
+import java.util.List;
+import view.LadderPrinter;
+import view.NamesPrinter;
+
+public class LadderGame {
+    private final Ladder ladder;
+    private final Names names;
+
+    public LadderGame(List<String> userNames, int ladderHeight) {
+        names = new Names(userNames);
+        int nameCount = names.getNameCount();
+        ladder = new Ladder(ladderHeight, nameCount, new BridgeRandomGenerator());
```

Comment:
헬퍼 메서드를 활용하면 LadderGame 내에서도 nameCount라는 값을 rowWidth 명확하게 표현할 수 있다고 생각해요!
크게 중요한 부분은 아닙니다 😃 

```suggestion
        ladder = new Ladder(ladderHeight, this.rowWidth(names), new BridgeRandomGenerator());
    }

    public String rowWidth(List<Names> names) {
        return this.names.getNameCount();
    }
```